showcase/app/styles/showcase-pages/side-nav.scss

```
@@ -8,6 +8,17 @@
 body.components-side-nav {
   $shw-component-side-nav-background: #0c0c0e;
 
+  // constrain max-width of button examples
+  .shw-component-button-examples-section {
+    .hds-side-nav:has(button) {
```

[suggestion/non-blocking] I don't think you need the `has()` here 🤔
```suggestion
    .hds-side-nav {
```

---

Thanks, you're right